liquidity/liquidity.go

```diff
@@ -327,15 +368,47 @@ func cloneParameters(params Parameters) Parameters {
 	return paramCopy
 }
 
+// autoloop gets a set of suggested swaps and dispatches them automatically if
+// we have automated looping enabled.
+func (m *Manager) autoloop(ctx context.Context) error {
+	swaps, err := m.SuggestSwaps(ctx, true)
```

Comment:
Not sure how "expensive" this call is, but maybe it makes sense to exit here already if auto loop is disabled?

Reply:
Also been thinking about this a bit, as is I have the autoloop logging the swaps that it _would_ have dispatched, which I think is nice. But it does cost us 1x quote call to the server per suggested swap, which could get costly. 

Since this is quite a new feature and you have to explicitly set a per-channel rule (which would result in a quote), WDYT of leaving it as is now to give some visibility into the functionality and then turning it off at a later stage?

Reply:
Sounds good to me :+1: